app/src/main/java/de/danoeh/antennapod/activity/WidgetConfigActivity.java

```diff
@@ -0,0 +1,96 @@
+package de.danoeh.antennapod.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.appwidget.AppWidgetManager;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.RelativeLayout;
+import android.widget.RemoteViews;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.preferences.UserPreferences;
+
+public class WidgetConfigActivity extends AppCompatActivity {
+
+    private static final String PREFS_WIDGET = "widget_preferences";
+    private static final String PREFS_COLOR = "widget_color";
+
+    private final int DEF_COLOR = 0x00262C31;
```

Comment:
Those string constants are already present in PlayerWidget. Please make them public there and reference them instead of copying them. Always try to have code in just one single location. If you need to copy-paste something, it will make it harder to maintain the code later.

Also, I think `DEFAULT_COLOR` instead of `DEF_COLOR` would be easier to read.